### PR TITLE
Fix: Update authentication guide to use .ts files instead of .mjs

### LINKED ISSUE #10218: auth-astro instructions show using a .mjs
### 📚 Subject area/topic

Guides

### 📋 Page(s) affected (or suggested, for new content)

https://docs.astro.build/en/guides/authentication/#configuration

### 📋 Description of content that is out-of-date or incorrect

The Astro docs show using a `.mjs` file for the auth config while the [official readme](https://github.com/nowaythatworked/auth-astro?tab=readme-ov-file#configuration) for the lib shows using a `.ts` file

This resulted in errors for a user of the library which can be found in [this support thread](https://discord.com/channels/830184174198718474/1314385854315368468/1314405117772173392) on the discord 

### 🖥️ Reproduction in StackBlitz (if reporting incorrect content or code samples)

_No response_

### PR DESCRIPTION
**Pull request by Kevin2606**
#### Description (required)

Updated the authentication guide to use `.ts` instead of `.mjs` for the configuration file. This aligns the documentation with the official library's README and prevents potential errors when users follow the guide.

#### Related issues & labels (optional)
- Closes #10218
- Suggested label: documentation

#### First-time contributor to Astro Docs?
Yes! 🎉
Discord: keenivg

### CODE REVIEW

**Review by apatel369 — APPROVED** [review]
Thanks for doing this @Kevin2606. Welcome to Team Docs! 🥳 

Your changes are perfect. 

Screenshot 2024-12-07 at 11 00 54 PM

File: src/content/docs/en/guides/authentication.mdx [hunk 1]
```diff
@@ -69,7 +69,7 @@ To install `auth-astro` manually, install the required package for your package
 
 Then, apply the integration to your `astro.config.*` file using the `integrations` property:
 
-```ts title="astro.config.mjs" ins={2,6}
+```ts title="astro.config.ts" ins={2,6}
```
Comment (yanthomasdev) [inline src/content/docs/en/guides/authentication.mdx:72]: Hey @Kevin2606, any reason the Astro config file was also changed to `.ts`? From my understanding only the `auth.config.*` file would need to be in TypeScript for it to be fixed? I am no expert in Auth.js, so let me know if I am wrong here!

Comment (Kevin2606) [reply src/content/docs/en/guides/authentication.mdx:72]: Hey @yanthomasdev, thanks for pointing that out! That change was actually a mistake on my part. The `astro.config` file doesn't need to be changed to `.ts`, and there was no reason to modify its extension. It should have remained as `astro.config.mjs`. I’ll revert that change right away.

Comment (apatel369) [reply src/content/docs/en/guides/authentication.mdx]: Oh, I missed it in my review. Thanks, Yan for catching it.

**Review by yanthomasdev — APPROVED** [review]
Thanks @Kevin2606, welcome to Team Docs!

**Pull request merged**